### PR TITLE
BZ1913075:  Improve the wording of exposed image registry route in UI (1.4.2)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1913075

### CODE REVIEW

**Comment by apinnick** [comment]
@XinRedhat Can you review this? Thanks

File: modules/migration-adding-cluster-to-cam.adoc [hunk 1]
```diff
@@ -11,20 +11,21 @@ You can add a cluster to the {mtc-full} ({mtc-short}) web console.
...
 .Procedure
 
 . Log in to the cluster.
-. Obtain the service account token:
+. Obtain a service account token:
 +
 [source,terminal]
 ----
-$ oc sa get-token migration-controller -n openshift-migration
+$ oc sa get-token <service_account> -n <namespace> <1>
```
Comment (XinRedhat) [inline modules/migration-adding-cluster-to-cam.adoc:26]: @apinnick I am curious why you don't use "migration-controller"  service account here?

**Comment by XinRedhat** [comment]
other looks good to me.

File: modules/migration-adding-cluster-to-cam.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-adding-cluster-to-cam.adoc:26]: @XinRedhat 
I wasn't sure whether this was one of those cases where the user could choose any service account with cluster-admin privileges. Does the user have to use migration-controller SA?

Comment (XinRedhat) [reply modules/migration-adding-cluster-to-cam.adoc:26]: so far yes. you can see the doc https://docs.openshift.com/container-platform/4.6/migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.html#migration-adding-cluster-to-cam_migrating-4-2-4

Comment (apinnick) [reply modules/migration-adding-cluster-to-cam.adoc:26]: Thanks. I'll revert here.

File: modules/migration-adding-cluster-to-cam.adoc [hunk 2]
```diff
@@ -11,15 +11,15 @@ You can add a cluster to the {mtc-full} ({mtc-short}) web console.
 
 .Prerequisites
 
-If you are using Azure snapshots to copy data:
+Optional: If you are using Azure snapshots to copy data:
```
Comment (kalexand-rh) [inline modules/migration-adding-cluster-to-cam.adoc:14]: It's not really optional. It's required if you use Azure, so you already had the restriction noted correctly. I'd revert this change.

File: modules/migration-adding-cluster-to-cam.adoc [hunk 3]
```diff
@@ -37,16 +37,17 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
 . Click *Add cluster*.
 . Fill in the following fields:
 
-* *Cluster name*: May contain lower-case letters (`a-z`) and numbers (`0-9`). Must not contain spaces or international characters.
-* *Url*: URL of the cluster's API server, for example, `\https://<master1.example.com>:8443`.
-* *Service account token*: String that you obtained from the source cluster.
-* *Exposed route to image registry*: Optional. You can specify a route to the image registry of your source cluster to enable direct migration for images, for example, `docker-registry-default.apps.cluster.com`.
+* *Cluster name*: The cluster name may contain lower-case letters (`a-z`) and numbers (`0-9`). It must not contain spaces or international characters.
```
Comment (kalexand-rh) [inline modules/migration-adding-cluster-to-cam.adoc:40]:
```suggestion
* *Cluster name*: The cluster name can contain lower-case letters (`a-z`) and numbers (`0-9`). It must not contain spaces or international characters.
```

> ! may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.

File: modules/migration-adding-cluster-to-cam.adoc [hunk 4]
```diff
@@ -37,16 +37,17 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
...
+* *URL*: Specify the API server URL, for example, `\https://<www.example.com>:8443`.
+* *Service account token*: Paste the `migration-controller` service account token.
+* *Exposed route host to image registry*: Optional: If you are using direct image migration, specify the exposed route to the image registry of the source cluster, for example, `www.example.apps.cluster.com`.
```
Comment (kalexand-rh) [inline modules/migration-adding-cluster-to-cam.adoc:43]:
```suggestion
* *Exposed route host to image registry*: If you are using direct image migration, specify the exposed route to the image registry of the source cluster, for example, `www.example.apps.cluster.com`.
```

If there's another optional use case for this parameter, please explain it, too.

File: modules/migration-adding-cluster-to-cam.adoc [hunk 5]
```diff
@@ -37,16 +37,17 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
...
 +
-Direct migration is much faster than migration with a replication repository.
+You can specify a port. The default port is `5000`.
 
-* *Azure cluster*: Optional. Select it if you are using Azure snapshots to copy your data.
-* *Azure resource group*: This field appears if *Azure cluster* is checked.
-* If you use a custom CA bundle, click *Browse* and browse to the CA bundle file.
+* *Azure cluster*: Optional: Select this option if you are using Azure snapshots to copy your data.
```
Comment (kalexand-rh) [inline modules/migration-adding-cluster-to-cam.adoc:47]:
```suggestion
* *Azure cluster*: You must select this option if you use Azure snapshots to copy your data.
```

File: modules/migration-adding-cluster-to-cam.adoc [hunk 6]
```diff
@@ -37,16 +37,17 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
...
+* *Azure resource group*: This field appears if *Azure cluster* is selected. Specify the Azure resource group.
+* *Require SSL verification*: Optional: Select this option to verify SSL connections to the cluster.
+* *CA bundle file*: This field appears if *Require SSL verification* is selected. Optional: If you created a custom CA certificate bundle file for self-signed certificates, click *Browse*, select the CA bundle file, and upload it.
```
Comment (kalexand-rh) [inline modules/migration-adding-cluster-to-cam.adoc:50]: Is this optional if you require SSL verification? If not:
```suggestion
* *CA bundle file*: This field is displayed if *Require SSL verification* is selected. If you created a custom CA certificate bundle file for self-signed certificates, click *Browse*, select the CA bundle file, and upload it.
```

**Review by kalexand-rh — COMMENTED** [review]
It's best to use "Optional" only if the entire parameter is always an option. If a parameter is required for a particular use case, it's better to explain when it's required.
